src/activeLighting.js

```diff
@@ -71,6 +71,21 @@ class ATL {
             default: false,
             type: Boolean,
         });
+        game.settings.register("ATL", "allowControlWithoutGm", {
+            name: "Allow Control without GM",
+            hint: "Allow users with this role and above to control ATL effects while the GM is not present (Requires reload)",
```

Comment:
Can you remove the reload suggestion? I think we should leave it up to Foundry or modules to indicate these now that Foundry can do the reload for us.
```suggestion
            hint: "Allow users with this role and above to control ATL effects while the GM is not present",
```

Reply:
Fixed in latest commit.

Reply:
I should note, I left the text on the hint to indicate to the user that it'll require a reload to take effect as they can still click to not reload, which won't do anything.